CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v3.0.1](https://github.com/voxpupuli/puppet-wildfly/tree/v3.0.1) (2023-10-27)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-wildfly/compare/v3.0.0...v3.0.1)
+
+**Merged pull requests:**
+
+- Update metadata.json [\#306](https://github.com/voxpupuli/puppet-wildfly/pull/306) ([rwaffen](https://github.com/rwaffen))
```

Comment:
This PR raise the upper bound of stdlib. We label such PRs as *enhancement*, then github-changelog-generator put them in a section with that name, and we release this as a new minor version (3.1.0).